pkgs/top-level/all-packages.nix

```diff
@@ -12273,7 +12273,9 @@ with pkgs;
   knot-dns = callPackage ../servers/dns/knot-dns { };
   knot-resolver = callPackage ../servers/dns/knot-resolver {
     # TODO: vimNox after it gets fixed on Darwin or something lighter
-    hexdump = if stdenv.isLinux then utillinux.bin else vim/*xxd*/;
+    hexdump = if stdenv.isLinux then utillinux.bin
```

Comment:
perhaps we should make this an actual top-level alias?

Reply:
:+1: for this. I believe it would work fine to just move this to top-level.

Oddly enough "vim" doesn't actually provide hexdump anymore (did it ever?). We do have an "xxd" derivation as well.

Reply:
I think some of the bitcoin derivations also need hexdump

Reply:
Yeah, maybe this build falls back to xxd or something.